### DISCUSSION TITLE
README and other docs localization

### DISCUSSION THREAD

**Discussion by TungusSs** [pear-devs/pear-desktop#719] [Ideas]
Hello! I have an idea about implementing localization in different languages in your application, or at least README. What do you think?

**Comment by inson1** [comment]
@TungusSs 
I created issue #1411  for Localization of the app
What about changing this discussion for only Readme Localization?

**Reply by TungusSs** [reply]
Hi, I didn't think that after so much time, someone would answer 😅 
I looked through the #1411 issue and the response from JellyBrick. I think it's really better to translate the README and the rest of the documentation for developers and translate the application itself via Hosted Webplate.
But I think we need to discuss this with the devs
@JellyBrick hey, what do you think about it?

**Reply by inson1** [reply]
@TungusSs Do you have some example? How it should looked and be done? I did see it just one time tbh, but I dont remember where

**Reply by TungusSs** [reply]
For example, the [nuclear](https://github.com/nukeop/nuclear) project, I translated the README and documentation for it. There, the translation is displayed in the form of national flags of the countries. Clicking on these flags opens the corresponding README. As for the structure of the project , I think it's worth making a separate docs folder in the root of the project repository and adding all files there.

**Reply by inson1** [reply]
@TungusSs What is the approach when main readme is changed?

**Reply by inson1** [reply]
@TungusSs And do you need something to do that? Or may you create PR for it? JellyBrick doesnt seem to mind or maybe even likes it.

**Comment by TungusSs** [comment]
> @TungusSs What is the approach when main readme is changed?

Unfortunately you'll have to edit it manually, but the README is supposed to change very rarely, especially for big projects like this one, so I guess it won't be a big problem.

**Comment by TungusSs** [comment]
> @TungusSs And do you need something to do that? Or may you create PR for it? JellyBrick doesnt seem to mind or maybe even likes it.

I just created a fork and a separate branch, did everything there and sent the PR. I don't know how it's done here exactly, but it's the same thing with PR anyway I think😄

**Reply by inson1** [reply]
@TungusSs Great work!

**Reply by TungusSs** [reply]
I meant that's what I did with the project. I haven't started working with that project yet. I'm still waiting for one of the developers to give permission to do that :)

**Comment by JellyBrick** [comment]
<a href="https://hosted.weblate.org/engage/youtube-music/">
  <img src="https://hosted.weblate.org/widget/youtube-music/i18n/multi-auto.svg" alt="translation status" />
  <img src="https://hosted.weblate.org/widget/youtube-music/i18n/287x66-black.png" alt="translation status 2" />
</a>

**Comment by JellyBrick** [comment]
Added multilingual README!

An example can be found here: https://github.com/th-ch/youtube-music/blob/master/docs/readme/README-ko.md
